### ISSUE TITLE
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: This url does not appear to be a playable track.

### ISSUE THREAD

**Issue by Sattishh** [sedmelluq/lavaplayer#651]
I was trying to play a song from Soundcloud when this showed up.

**Comment by MCOfficer** [comment]
Please provide the URL you were trying to play, your lavaplayer version, and the full stacktrace.

**Comment by Sattishh** [comment]
URL : https://soundcloud.com/twinteefineohfor/subway-surfers-lb66

Lavaplayer version : 1.3.78

```
Caused by: java.lang.RuntimeException: No track JSON found EID: f13367e7-ce62-43ad-b92f-82f71b662689, html <redacted>
        at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.throwWithDebugInfo(ExceptionTools.java:134) ~[lavaplayer-1.3.78.jar!/:na]
        ... 11 common frames omitted

2021-08-14 03:36:37.221 ERROR 20504 --- [ader-2-thread-1] l.server.player.AudioLoaderRestHandler   : Track loading failed

com.sedmelluq.discord.lavaplayer.tools.FriendlyException: This url does not appear to be a playable track.
        at com.sedmelluq.discord.lavaplayer.source.soundcloud.DefaultSoundCloudHtmlDataLoader.load(DefaultSoundCloudHtmlDataLoader.java:43) ~[lavaplayer-1.3.78.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.soundcloud.SoundCloudAudioSourceManager.loadFromTrackPage(SoundCloudAudioSourceManager.java:202) ~[lavaplayer-1.3.78.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.soundcloud.SoundCloudAudioSourceManager.processAsSingleTrack(SoundCloudAudioSourceManager.java:179) ~[lavaplayer-1.3.78.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.source.soundcloud.SoundCloudAudioSourceManager.loadItem(SoundCloudAudioSourceManager.java:116) ~[lavaplayer-1.3.78.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:436) ~[lavaplayer-1.3.78.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:418) ~[lavaplayer-1.3.78.jar!/:na]
        at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:217) ~[lavaplayer-1.3.78.jar!/:na]
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
        at java.base/java.lang.Thread.run(Thread.java:829) ~[na:na]
        Suppressed: com.sedmelluq.discord.lavaplayer.tools.exception.EnvironmentInformation:
  lavaplayer.version: 1.3.78
  os.arch: amd64
  os.name: Linux
  os.version: 4.15.0-51-generic
  java.vendor: Ubuntu
  java.version: 11.0.11
  java.runtime.version: 11.0.11+9-Ubuntu-0ubuntu2.18.04
  java.vm.version: 11.0.11+9-Ubuntu-0ubuntu2.18.04
Caused by: java.lang.RuntimeException: No track JSON found EID: f13367e7-ce62-43ad-b92f-82f71b662689, html <redacted>
        at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.throwWithDebugInfo(ExceptionTools.java:134) ~[lavaplayer-1.3.78.jar!/:na]
        ... 11 common frames omitted


```

**Comment by MCOfficer** [comment]
This may be fixed by #650, but i can't tell for sure. @NavyCake ?

**Comment by nayvcake** [comment]
> This may be fixed by #650, but i can't tell for sure. @NavyCake ?

@MCOfficer Yes, with these changes I made. it's working.

Playlist:
![Cg3UkoHSxW](https://user-images.githubusercontent.com/43297329/129452661-df1f7ec3-620e-4a18-ba96-4d423e5bed35.png)

Track:
![OmVB9tIT7n](https://user-images.githubusercontent.com/43297329/129452899-80da5a22-8b37-4df4-9a78-53e71833d538.png)

**Comment by Sattishh** [comment]
Alright let's wait for the pr to be accepted
